starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items  = [];
+    this.length = 0;
+  }
+  add(x){
+    this.items.push(x);
+    this.length++;
+  }
+
+  get(x){
+    if(this.items.length=== 0){
+      throw new Error("EmptySortedList")}
+    if(x > this.length){
+      throw new Error("OutOfBounds")}
+    return this.items[x];
+}
+
+  max(items){
+    if(this.items.length=== 0){
+      throw new Error("EmptySortedList")
+    }
+    return Math.max(...items);
+  }
+
+  min(items){
+    if(this.items.length=== 0){
+      throw new Error("EmptySortedList");
+    }
+    return Math.min(...items);
+  }
+
+  sum(items){
+    this.items.reduce(acc, curr)
```

Comment:
you're close with the sum function, but you're missing a few things from your reduce. You never turned it into a function. 

```
  sum() {
    return this.items.reduce(function(acc, curr) {
      return acc + curr;
    }, 0);
  }
```